scripts/generate.py

```diff
@@ -50,7 +50,8 @@ def _gen_class_name(path: str) -> str:
         return name
 
     typ_classes = {}
-    for typ, paths in typ_paths.items():
+    sorted_typ_paths = sorted(typ_paths.items(), key=lambda x:x[0])
```

Comment:
In Python 3.6 or higher, we can sort the dict by just `sorted(typ_paths.items())`. We don't need the custom key function. 

`for typ, paths in sorted(typ_paths.items()):` is enough I think.

Reply:
Thank you advice. I Fixed it.